Lib/test/test_math.py

```diff
@@ -2033,6 +2033,12 @@ def testComb(self):
             self.assertIs(type(comb(IntSubclass(5), IntSubclass(k))), int)
             self.assertIs(type(comb(MyIndexable(5), MyIndexable(k))), int)
 
+    def test_nextafter(self):
+        self.assertEqual(math.nextafter(9223372036854775807.0, 0.0),
```

Comment:
It would be clearer to use a value that's exactly representable here: `9223372036854775808.0` rather than `9223372036854775807.0`. Otherwise there are two effects going on: first, the literal is being rounded to a different value, and *then* that value is having `nextafter` applied to it.